BUILD-WINDOWS.md

```diff
@@ -96,24 +96,19 @@ Ruby is needed both for a number of the build steps and as the main
 runtime for the language server. We need to install both it and some
 additional libraries.
 
-Firstly, install the latest version of Ruby (2.7.1 - 64 bit with devkit) from:
+Firstly, install the latest version of Ruby (3.1.0 - 64 bit with devkit) from:
 
-https://github.com/oneclick/rubyinstaller2/releases/download/RubyInstaller-2.7.1-1/rubyinstaller-devkit-2.7.1-1-x64.exe
+https://github.com/oneclick/rubyinstaller2/releases/download/RubyInstaller-3.1.0-1/rubyinstaller-devkit-3.1.0-1-x64.exe
 
 Once you have installed Ruby, you need to grab some additional
 libraries. We can do this from the command prompt with the following:
 
 ```
-gem install win32-process
-gem install rugged --version 0.27.1
-gem install ffi
+gem install rugged
 ```
 
-This uses the Ruby library management tool `gem` to install 3
-dependencies for us - `win32-process` used to help manage Windows OS
-processes, rugged which is used to store the code diffs in a local Git
-repo and ffi which is used to hook into external shared libs such as
-libaubio.
+This uses the Ruby library management tool `gem` rugged which is used to
```

Comment:
This edit seems a bit mixed up to me - is it correct?

Reply:
Good catch, I've fixed it now :-)